src/platform/linux/iotjs_module_gpio-linux.c

```diff
@@ -196,6 +201,19 @@ static bool gpio_set_edge(iotjs_gpio_t* gpio) {
 }
 
 
+void iotjs_gpio_platform_create(iotjs_gpio_t_impl_t* _this) {
+  size_t private_mem = sizeof(struct _iotjs_gpio_module_platform_t);
+  _this->platform = (iotjs_gpio_module_platform_t)malloc(private_mem);
+  _this->platform->value_fd = -1;
+  uv_mutex_init(&_this->platform->mutex);
+}
+
+
+void iotjs_gpio_platform_destroy(iotjs_gpio_t_impl_t* _this) {
+  free(_this->platform);
```

Comment:
Please use `iotjs_buffer_release` to check if `_this->platform` is `NULL`. Or you should 
 initialize `_this->platform` to `NULL` after free it.